### ISSUE TITLE
[BUG] "Companion is unable to start" on Windows 10 

### ISSUE THREAD

**Issue by theomnipex** [bitfocus/companion#2885]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

We have recently updated to the newest Version of Companion. Rebooted and now when we try to start the application, the message pops up and companion closes. 
![Screenshot 2024-05-27 103351](https://github.com/bitfocus/companion/assets/170927957/a10954e9-3bd1-46f0-86ab-06878d35d512)

### Steps To Reproduce

1. Try to run companion 
2. Error message pops-up

### Expected Behavior

Companion should start normally as before.

### Environment (please complete the following information)

```markdown
- OS: Windows 10 
- Browser: Microsoft Edge 64-bit
- Companion Version: v3.3.0+6991-stable-b27baf94
```

### Additional context

Find the log attached here
[companion-2024-05-28.0.log](https://github.com/bitfocus/companion/files/15465533/companion-2024-05-28.0.log)

**Comment by Julusian** [comment]
This looks to be caused by https://github.com/automation-stack/node-machine-id/issues/56, invoked at https://github.com/bitfocus/companion/blob/main/companion/lib/Data/CloudDatabase.js#L32

So I guess we need some fallback mechanism to handle environments where windows is more locked down

**Comment by Julusian** [comment]
This is fixed in the betas and will be in a 3.3.1 patch release 'soon'
